apps/ttalps_histogrammer.cpp

```diff
@@ -69,10 +72,12 @@ int main(int argc, char **argv) {
   for (int iEvent = 0; iEvent < eventReader->GetNevents(); iEvent++) {
     auto event = eventReader->GetEvent(iEvent);
 
+    ttalpsObjectsManager->InsertMuonTriggerCollections(event);
     ttalpsObjectsManager->InsertMatchedLooseMuonsCollections(event);
     ttalpsObjectsManager->InsertMuonVertexCollection(event);
     ttalpsObjectsManager->InsertNminus1VertexCollections(event);
     ttalpsObjectsManager->InsertBaseLooseMuonVertexCollection(event);
+    if (runNonLeadingVertexCollection) ttalpsObjectsManager->InsertNonLeadingMuonVertexCollection(event);
```

Comment:
Yeah, I guess I'm still confused how this relates to `BestPFIsoDimuonVertex`, `BestDimuonVertex`, `SRDimuonVertex`, `JPsiDimuonVertex`, etc., and especially why this one is treated differently.